listenbrainz/webserver/static/css/follow.less

```diff
@@ -75,3 +82,12 @@
     }
   }
 }
+
+.follower-following-empty, .similar-users-empty {
+  padding: 10px 20px;
+}
+
+hr {
+  margin: 0px 2em;
+  border-top: 1px solid #eee;
+}
```

Comment:
This now applies to all `hr` elements across the website. 
The fact that we're in a separate .less file makes no difference to that, in the end everything is compiled into a single css file.

If you want it to apply only to hr inside certain classes, you can move this whole block inside of it like this:
```less
.follower-following-empty, .similar-users-empty {
  padding: 10px 20px;
  hr {
    margin: 0px 2em;
    border-top: 1px solid #eee;
  }
}
```

This will be compiled as 
```less
.follower-following-empty hr,  .similar-users-empty hr {…}
```
This nesting capability comes from LESS, it's not a native CSS feature.

Reply:
Oh right! I forgot it compiles to a single css file. I'll work on it.